storage/bucket_test.go

```diff
@@ -121,6 +122,7 @@ func TestBucketAttrsToRawBucket(t *testing.T) {
 			UniformBucketLevelAccess: &raw.BucketIamConfigurationUniformBucketLevelAccess{
 				Enabled: true,
 			},
+			PublicAccessPrevention: "enforced",
```

Comment:
Can "enforced" be declared as a constant and then used everywhere?

Reply:
I think this is not super typical in Go tests; if you look above you see that most other checks in this test use explicit strings.